_layouts/eip.html

```diff
@@ -18,14 +18,29 @@
 </svg>
 
 <div class="home">
+  <h1 class="page-heading">
+    {% if page.category == "ERC" %}ERC{% else %}EIP{% endif -%}-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
+    <a href="{{ page["discussions-to"] | uri_escape }}" class="no-underline">
```

Comment:
Hadn't we switched this to:

```suggestion
    <a href="{{ page.discussions-to | uri_escape }}" class="no-underline">
```